src/main/resources/messages_vi_VN.properties

```diff
@@ -0,0 +1,1053 @@
+addPageNumbers.customTextDesc=Văn bản tùy chỉnh
+addPageNumbers.numberPagesDesc=Những trang cần đánh số, mặc định là 'all', cũng chấp nhận 1-5 hoặc 2,5,9 v.v.
+addPageNumbers.customNumberDesc=Mặc định là {n}, cũng chấp nhận 'Trang {n} / {total}', 'Văn bản-{n}', '{filename}-{n}
+addPageNumbers.submit=Thêm số trang
+
+#auto-rename
+auto-rename.title=Tự động đổi tên
+auto-rename.header=Tự động đổi tên PDF
+auto-rename.submit=Tự động đổi tên
+
+#adjustContrast
+adjustContrast.title=Điều chỉnh độ tương phản
+adjustContrast.header=Điều chỉnh độ tương phản
+adjustContrast.contrast=Độ tương phản:
+adjustContrast.brightness=Độ sáng:
+adjustContrast.saturation=Độ bão hòa:
+adjustContrast.download=Tải xuống
+
+#crop
+crop.title=Cắt
+crop.header=Cắt PDF
+crop.submit=Gửi
+
+#autoSplitPDF
+autoSplitPDF.title=Tự động chia PDF
+autoSplitPDF.header=Tự động chia PDF
+autoSplitPDF.description=In, chèn, quét, tải lên và để chúng tôi tự động tách tài liệu của bạn. Không cần sắp xếp thủ công.
+autoSplitPDF.selectText.1=In một số trang phân cách từ bên dưới (Đen trắng là được).
+autoSplitPDF.selectText.2=Quét tất cả tài liệu của bạn cùng một lúc bằng cách chèn trang phân cách giữa chúng.
+autoSplitPDF.selectText.3=Tải lên tệp PDF quét lớn duy nhất và để Stirling PDF xử lý phần còn lại.
+autoSplitPDF.selectText.4=Các trang phân cách được tự động phát hiện và xóa, đảm bảo tài liệu cuối cùng gọn gàng.
+autoSplitPDF.formPrompt=Gửi PDF chứa trang phân cách Stirling-PDF:
+autoSplitPDF.duplexMode=Chế độ hai mặt (Quét mặt trước và sau)
+autoSplitPDF.dividerDownload1=Tải xuống 'Trang phân cách tự động (tối giản).pdf'
+autoSplitPDF.dividerDownload2=Tải xuống 'Trang phân cách tự động (có hướng dẫn).pdf'
+autoSplitPDF.submit=Gửi
+
+#pipeline
+pipeline.title=Pipeline
+
+#pageLayout
+pageLayout.title=Bố cục nhiều trang
+pageLayout.header=Bố cục nhiều trang
+pageLayout.pagesPerSheet=Số trang trên một tờ:
+pageLayout.addBorder=Thêm viền
+pageLayout.submit=Gửi
+
+#scalePages
+scalePages.title=Điều chỉnh tỷ lệ trang
+scalePages.header=Điều chỉnh tỷ lệ trang
+scalePages.pageSize=Kích thước của một trang trong tài liệu.
+scalePages.scaleFactor=Mức độ phóng to (cắt) của một trang.
+scalePages.submit=Gửi
+
+#certSign
+certSign.title=Ký bằng chứng chỉ
+certSign.header=Ký PDF bằng chứng chỉ của bạn (Đang trong quá trình phát triển)
+certSign.selectPDF=Chọn tệp PDF để ký:
+certSign.jksNote=Lưu ý: Nếu loại chứng chỉ của bạn không được liệt kê bên dưới, vui lòng chuyển đổi nó thành tệp Java Keystore (.jks) bằng công cụ dòng lệnh keytool. Sau đó, chọn tùy chọn tệp .jks bên dưới.
+certSign.selectKey=Chọn tệp khóa riêng của bạn (định dạng PKCS#8, có thể là .pem hoặc .der):
+certSign.selectCert=Chọn tệp chứng chỉ của bạn (định dạng X.509, có thể là .pem hoặc .der):
+certSign.selectP12=Chọn tệp Keystore PKCS#12 của bạn (.p12 hoặc .pfx) (Tùy chọn, nếu cung cấp, nó phải chứa khóa riêng và chứng chỉ của bạn):
+Here's the continuation of the translation from "certSign.selectP12":
```

Comment:
Please remove...
More LLM stuff here..... Do you have confidence in these translations?